.github/workflows/push-mimir-build-image.yml

```diff
@@ -0,0 +1,108 @@
+name: Build and Push mimir-build-image
+
+# configure trigger by pull request or manually
+on:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - mimir-build-image/Dockerfile  
```

Comment:
Should this be `mimir-build-image/*`, in case we want to be able to add more files to the image? If so, we'll need to adjust the checksum calculation below as well.

Reply:
For now we have only the case for dockerfile (go, alpine update), wdyt let's try this first, if we see include repo is necessary, we can change that easily.

Reply:
Sure, sounds good to me.